### ISSUE TITLE
Cant see division of matrix by sqrt(dk)

### ISSUE THREAD

**Issue by Kikumu** [tunz/transformer-pytorch#8]
https://github.com/tunz/transformer-pytorch/blob/e7266679f0b32fd99135ea617213f986ceede056/model/transformer.py#L87

I noticed that according to the paper  the query and key values are then divided by d_k before passing to softmax. I dont see it in the code, did i miss anything? Thank you!

**Comment by tunz** [comment]
https://github.com/tunz/transformer-pytorch/blob/e7266679f0b32fd99135ea617213f986ceede056/model/transformer.py#L84

It's divided by `d_k` here. `self.scale` is `1/d_k`.

**Comment by Kikumu** [comment]
Aaah got it - thanks!
